packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/redact/requests/UnredactRequest.java

```diff
@@ -0,0 +1,41 @@
+package cloud.pangeacyber.pangea.redact.requests;
+
+import cloud.pangeacyber.pangea.BaseRequest;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class UnredactRequest extends BaseRequest {
+
+	@JsonProperty("redacted_data")
+	Object redactedData;
```

Comment:
Could be generic? See `EncryptStructuredRequest` for example.

Reply:
Done